.bazelrc

```diff
@@ -13,6 +13,12 @@ common --workspace_status_command "${PWD}/workspace_status.sh"
 
 common:release -c opt --stamp
 
+# --noexperimental_check_output_files setting from .aspect/bazelrc/performance.bazelrc breaks "build without the bytes" with Bazel 7
```

Comment:
lots of users will need this change in their own repo as well, right?

Reply:
If they are using "build without the bytes" with bazel and our recommended settings and setting `remote_download_outputs=toplevel` then yes. I don't think that is a very common intersection however

Reply:
ha, happened to come back here from the thread with @pyrocat101